print-engine/src/main/java/de/tillhub/printengine/html/HtmlParams.kt

```diff
@@ -1,21 +1,21 @@
 package de.tillhub.printengine.html
 
-internal enum class FeedString(val value: String) {
+enum class FeedString(val value: String) {
```

Comment:
We need different aproach here, because print-engine is not aware of Pax or Verifone. You can keep naming but change them into value class and pass values from addons